### PR TITLE
Translations: Add Korean to language selection

### PR DESCRIPTION
**Pull request by hoffie**
**Short description of changes**

- Translations: Make ko-KR (Korean) selectable
- Jamulus.pro: Add reminder for new translations (qrc)

CHANGELOG: Translation: Added Korean translation to Jamulus

**Context: Fixes an issue?**

Fixes: #2740 #2714
Related: #2685

**Does this change need documentation? What needs to be documented and how?**

**Status of this Pull Request**

Ready.

**What is missing until this pull request can be merged?**

Reviews.

@bagjunggyu Could you check the release artifacts from this PR?

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

**Comment by bagjunggyu** [comment]
Surely!
Let me check this out with other users.
Thank you :)

File: src/resources.qrc [hunk 1]
```diff
@@ -32,6 +32,9 @@
 	<qresource prefix="/translations">
         <file alias="translation_zh_CN">translation/translation_zh_CN.qm</file>
     </qresource>
+	<qresource prefix="/translations">
```
Comment (ann0see) [inline src/resources.qrc:35]: Strange. I thought I did that... nevertheless thanks.

**Review by ann0see — APPROVED** [review]
Looks good. However waiting for test by @bagjunggyu

**Comment by bagjunggyu** [comment]
I don't have any idea for build issue though.

If it is OK with 3.8.2dev win.exe file done by auto build several hours ago.

I've got message from Korean user and it is affected well as this image.
56151201-C074-4135-89DF-317F90E4A94C

it has affected well!
Thank you :)

**Comment by hoffie** [comment]
@bagjunggyu Thanks for confirming!
The autobuild failure (Windows) is unrelated and is caused by flakiness of one of the dependency downloads. The test with the older build is fine, I've just force-pushed ~2 hours ago due to a whitespace issue in the XML file.

**Comment by pljones** [comment]
@hoffie @ann0see Is there a guide on adding a new translation and does it include this step?

**Comment by pljones** [comment]
Given we've just done it for Korean, it might be worth writing one now before the memories fade away.

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
@bagjunggyu can you edit The TRANSLATION.md file in the docs/ folder and write up a short paragraph what needs to be done (you can pair up with @gilgongo (our "documentation guy")

**Comment by bagjunggyu** [comment]
As I said before, it is hard for me to connect to internet and do something with electric devices.
I'm afraid that it would not be made.

**Pull request merged**